contributing.md

```diff
@@ -137,6 +137,13 @@ python scripts/dev/e2e.py --test <new-test>
 
 # Before Committing your code
 
-Please make sure you sign our Contributor Agreement
-[here](https://www.mongodb.com/legal/contributor-agreement). This will be
+## Set up pre-commit hooks
+To set up the pre-commit hooks, please create symbolic links from the provided [hooks](https://github.com/mongodb/mongodb-kubernetes-operator/tree/master/scripts/git-hooks):
+
+1. Navigate to your `.git/hooks` directory: `cd .git/hooks`
+1. Create a symlink for every file in the `scripts/git-hooks` directory: `ln -s -f  ../../scripts/git-hooks/* .`
```

Comment:
[nit] I think it would be nice to have each command on its own line, so the full line is copy paste-able. Not a huge deal of course!